### PR TITLE
docs: add forge script flags etherscan-api-key, json, verify

### PR DESCRIPTION
**Pull request by shawnharmsen**
closes: https://github.com/foundry-rs/book/issues/357

### CODE REVIEW

File: src/reference/forge/forge-script.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ forge-script - Run a smart contract as a script, building transactions that can
 
 ### DESCRIPTION
 
+> Note: The output is under development and prone to change.
```
Comment (onbjerg) [inline src/reference/forge/forge-script.md:13]:
```suggestion

```

File: src/reference/forge/forge-script.md [hunk 2]
```diff
@@ -38,6 +40,16 @@ Scripts can be used to apply state transitions on live contracts, or deploy and
 `--target-contract` *contract_name*  
 &nbsp;&nbsp;&nbsp;&nbsp;The name of the contract you want to run.
 
+`--etherscan-api-key` *key*  
+&nbsp;&nbsp;&nbsp;&nbsp;Etherscan API key.  
+&nbsp;&nbsp;&nbsp;&nbsp;Environment: `ETHERSCAN_API_KEY`
+
+`--json`  
+&nbsp;&nbsp;&nbsp;&nbsp;Output results in JSON format.
```
Comment (onbjerg) [inline src/reference/forge/forge-script.md:48]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;Output results in JSON format.  
&nbsp;&nbsp;&nbsp;&nbsp;Note: The output is under development and prone to change.
```

**Review by onbjerg — CHANGES_REQUESTED** [review]
smol nits

File: src/reference/forge/forge-script.md [hunk 3]
```diff
@@ -38,6 +40,16 @@ Scripts can be used to apply state transitions on live contracts, or deploy and
...
+
+`--verify`
```
Comment (Abirhasankas143) [inline src/reference/forge/forge-script.md:50]: Info

**Review by Abirhasankas143 — APPROVED** [review]

File: src/reference/forge/forge-script.md [hunk 4]
```diff
@@ -38,6 +39,17 @@ Scripts can be used to apply state transitions on live contracts, or deploy and
...
 
+`--etherscan-api-key` *key*  
+&nbsp;&nbsp;&nbsp;&nbsp;Etherscan API key.  
+&nbsp;&nbsp;&nbsp;&nbsp;Environment: `ETHERSCAN_API_KEY`
+
+`--json`  
+&nbsp;&nbsp;&nbsp;&nbsp;Output results in JSON format.  
+&nbsp;&nbsp;&nbsp;&nbsp;Note: The output is under development and prone to change.
+
+`--verify`
```
Comment (onbjerg) [inline src/reference/forge/forge-script.md:50]:
```suggestion
`--verify`  
```

**Review by onbjerg — APPROVED** [review]

**Pull request merged**